### PR TITLE
Fix: Run sanity and integration tests with kind

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: test/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml [hunk 1]
```diff
@@ -47,11 +47,14 @@ spec:
             - containerPort: {{ .Values.controller.metricsPort }}
               name: metrics
               protocol: TCP
+            - containerPort: 10000
```
Comment (edreed) [inline test/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml:50]: This should be `{{ .Values.controller.port }}`

File: test/latest/azuredisk-csi-driver/values.yaml [hunk 2]
```diff
@@ -13,9 +13,11 @@ azuredisk:
   useV2Driver: true
   csiEndpoint: tcp://127.0.0.1:10000
```
Comment (edreed) [inline test/latest/azuredisk-csi-driver/values.yaml:14]: I believe we can remove this value now.

**Review by edreed — APPROVED** [review]
A couple of minor changes needed; otherwise, LGTM.

File: test/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml (hunk 1 above)
Comment (nearora-msft) [reply test/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml]: Fixed

**Pull request merged**